guides/common/modules/snip_deprecated-feature.adoc

```diff
@@ -3,7 +3,7 @@
 
 [IMPORTANT]
 ====
-The {FeatureName} is a deprecated feature.
+{FeatureName} is a deprecated feature.
```

Comment:
I wonder if we should adjust line 6 in `guides/common/modules/proc_registering-hosts-by-using-the-bootstrap-script.adoc` to write "The bootstrap script ...". Your call.

Reply:
Yes! I must have missed that. Thanks.